book/website/history.md

```diff
@@ -0,0 +1,43 @@
+# History of The Turing Way project
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
+Dr. Whitaker brought together a team of [allies and contributors](https://github.com/the-turing-way/the-turing-way/blob/main/ways_of_working.md#previous-project-members), who were already championing reproducible practices in data science, to co-develop the first version of *The Turing Way* book.
+The project team included researchers, librarians, data scientists and Research Engineers, from both The Alan Turing Turing and the institutes across the UK.
+The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working open.
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as Collaboration Cafe, Book Dash).
+In-person workshops were given in person on tools aligned with the project: particulary Binder and JupterBook, and the project was presented in many conferences across the UK and internationally.
+
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
+As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
```

Comment:
This really needs to be more nuanced about how Open Life Science AND The Turing Way are inspired by the Mozilla Open Leadership training programme.

Both @malvikasharan and I credit Mozilla Open Leadership with transformational mentorship that has led us to contribute to changing the world through open and inclusive science practices.

OLS and TTW had parallel development rather than one coming before the other... I don't think OLS needs to be specifically mentioned in the history of The Turing Way project page.

@malvikasharan - what do you think?

[EDIT: I'm in a call with Malvika now - she agrees 😅]

Reply:
Very true! I think this was something that was visible to me when I joined the team - but might be confusing for people who are just looking to learn about the project's history in a specific, contained way!